app/controllers/katello/api/v2/content_views_controller.rb

```diff
@@ -20,6 +20,31 @@ class Api::V2::ContentViewsController < Api::V2::ApiController
       param :component_ids, Array, :desc => N_("List of component content view version ids for composite views")
     end
 
+    api :POST, "/content_views/:id/export", N_("Export a content view to disk")
+    param :id, :identifier, :desc => N_("Content view identifier"), :required => true
```

Comment:
I think you actually mean content_view_version, as that contains the 'snapshot' of all the repos.